src/components/widgets/EditableDate.vue

```diff
@@ -83,10 +82,9 @@ function formatDate(date: Date | string): string {
 function parseDate(dateString: string): Date {
   return DateTime.fromFormat(dateString, formatString, { zone: 'utc' }).toJSDate();
 }
-
 const maskState = reactive({
   completed: false,
-  masked: '',
+  masked: ' ',
```

Comment:
This little trick solves the problem of having the cursor at the end of the input on first focus
![image](https://github.com/RedHatProductSecurity/osim/assets/4268580/c539d798-7a3f-4488-995a-52ff21d758b9)